core/3-binary-data/project.lock.json

```diff
@@ -170,12 +170,12 @@
           "Zlib.Portable.Signed": "1.11.0"
         },
         "compile": {
-          "lib/net45/Google.Apis.dll": {},

```

Comment:
Why are there updates to the dependencies along with this?  Did you do a NuGet update?  Otherwise I'm :+1:!